packages/user/src/components/Invitation/InvitationsTable.tsx

```diff
@@ -44,6 +44,7 @@ export type InvitationsTableProperties = {
   prepareInvitationData?: (data: any) => any;
   roles?: Array<InvitationRoleOption>;
   showAppColumn?: boolean;
+  showFilterSearch?: boolean;
```

Comment:
@Nabin99 should we use `filterableColumns: FilterableColumn[]` like visibleColumns? what do you think? If we want this way, maybe we can change it to `showEmailFilter`.